src/types.rs

```diff
@@ -204,10 +204,26 @@ impl From<HWIStatus> for Result<(), Error> {
         }
     }
 }
+
 pub enum LogLevel {
     DEBUG,
     INFO,
     WARNING,
     ERROR,
     CRITICAL,
 }
+
+#[derive(Clone, Eq, PartialEq, Debug, Copy)]
+#[repr(u8)]
+/// The number of words in the recovery phrase
+pub enum HWIWordCount {
+    W12 = 12,
+    W18 = 18,
+    W24 = 24,
+}
+
+impl HWIWordCount {
+    pub fn as_u8(&self) -> u8 {
+        *self as u8
+    }
+}
```

Comment:
Is this needed? Can't you just do `wordcount as u8`?

I tried this in local and it seemed to work:
```diff
diff --git a/src/interface.rs b/src/interface.rs
index 916d884..e34ee8d 100644
--- a/src/interface.rs
+++ b/src/interface.rs
@@ -436,7 +436,7 @@ impl HWIClient {
         word_count: Option<HWIWordCount>,
     ) -> Result<(), Error> {
         Python::with_gil(|py| {
-            let word_count: u8 = word_count.map_or_else(|| 24, |w| w.as_u8());
+            let word_count = word_count.map_or_else(|| 24, |w| w as u8);
             let func_args = (&self.hw_client, label.unwrap_or(""), word_count);
             let output = self
                 .hwilib
diff --git a/src/types.rs b/src/types.rs
index 5bc7cd8..d652880 100644
--- a/src/types.rs
+++ b/src/types.rs
@@ -234,9 +234,3 @@ pub enum HWIWordCount {
     W18 = 18,
     W24 = 24,
 }
-
-impl HWIWordCount {
-    pub fn as_u8(&self) -> u8 {
-        *self as u8
-    }
-}
```